### PR TITLE
Prototype

### PR DESCRIPTION
**Pull request by lukehedger**
Adds example use of `meta-id` as outlined in #1

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+  const downloadButton = document.querySelector('.downloadKeystore')
+  downloadButton.setAttribute('href', url)
+  downloadButton.setAttribute('download', filename)
+
+  // render public address
+  document.querySelector('.ethereumAddress').textContent = account.address
+
+  // set connected account to newly created account
+  state.currentAccount = account
+}
```
Comment (MrTibbles) [inline example/index.js]: I am intrigued how web3 handles the unlocking of this newly created account on the underlying ETH instance. Unless it simply creates the account with password set as an empty string, worth looking into for sure.

File: example/index.js [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+  // get currently connected account
+  const account = getState('currentAccount')
+
+  // send funds from coinbase to connected account
+  return web3.eth.getAccounts().then(accounts => accounts[4]).then(coinbase =>
+    web3.eth.sendTransaction({
+      from: coinbase,
+      gas: 4700000,
+      to: account.address,
+      value: 999999999999999999,
```
Comment (MrTibbles) [inline example/index.js]: 💵

File: example/index.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+          <input class="verifyMetaIdName" type="text" value="" placeholder="META-ID">
+          <input class="verifyMetadataKey" type="text" value="aux/twitter" placeholder="aux/twitter">
+        </form>
+
+        <button class="verifyIdentity">Verify Metadata</button>
+      </div>
+    </div>
+    <script src="./bundle.js"></script>
+  </body>
+</html>
```
[the remark this answers is not in the document]
Comment (lukehedger) [reply example/index.html]: Have added comments

**Comment by lukehedger** [comment]
Once the event subscriptions are added the console will display both success and (albeit vague) error messages. Then once the argument validation #3 has been added the error messages will become more helpful. I think UX/styling can be added when needed.

**Comment by MrTibbles** [comment]
Thanks for clarifying.

> I think UX/styling can be added when needed.

Agreed, no rush.

**Pull request merged**